lightning/src/util/events.rs

```diff
@@ -122,6 +123,12 @@ pub enum Event {
 		/// Note that this serves as a payment receipt, if you wish to have such a thing, you must
 		/// store it somehow!
 		payment_preimage: PaymentPreimage,
+		/// The hash which was given to [`ChannelManager::send_payment`]. When the `PaymentSent` message
+		/// comes from an older library version, the payment hash will be missing, in which case the
```

Comment:
The latter part of the documentation here should be dropped.

Reply:
Fixed.